src/commands/exportApi.ts

```diff
@@ -49,4 +57,33 @@ export namespace ExportAPI {
         await vscode.workspace.fs.writeFile(vscode.Uri.file(localFilePath), Buffer.from(fileContent));
         await vscode.window.showTextDocument(document);
     }
+
+    async function getDefinitionTreeNode(context: IActionContext): Promise<ApiVersionDefinitionTreeItem | null> {
```

Comment:
In our scope for the first release of data plane, do we have other commands that will trigger from Command Palette?

Even though ExportAPI is the only one for first release, we'd better to put this shared method in another file that all command could reuse.

Reply:
updated.